doc/sphinx/user/install/docker-container/running-models.md

```diff
@@ -22,7 +22,7 @@ recommend the following workflow:
     you have started the container run the aspect model inside the container.
     Note that there are two ASPECT executables
     in the work directory of the container: `aspect` and `aspect-release`. For
-    a discussion of the different versions see {ref}`4.4][], in
+    a discussion of the different versions see {ref}4.4][], in
```

Comment:
How do we want to deal with this going forward? We can't reasonably have numbers be the labels for sections.